config/default.json

```diff
@@ -49,5 +49,31 @@
     "session": "blf-alpha-session"
   },
   "hotjarId": 1036292,
-  "googleAnalyticsCode": "UA-98908627-1"
+  "googleAnalyticsCode": "UA-98908627-1",
+  "archivedPaths": {
```

Comment:
Rather than living in the config could this all go in `modules/archived`?

Reply:
Yeah good call – take a look now.